app/cli/cmd/credits.go

```
@@ -0,0 +1,58 @@
+	}
+
+	if !org.IntegratedModelsMode {
+		fmt.Println("The org isn't using integrated models mode.")
+		return
+	}
+
+	balance := org.CloudBillingFields.CreditsBalance
+	balanceStr := fmt.Sprintf("$%s", balance.StringFixed(4))
+
+	for i := 0; i < 2; i++ {
+		if strings.HasSuffix(balanceStr, "0") {
```

golangci-lint (1.64.8)

S1017: should replace this if statement with an unconditional strings.TrimSuffix

(gosimple)